packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -116,8 +129,9 @@ class CupertinoAlertDialog extends StatelessWidget {
     Key key,
```

Comment:
In the docs for the constructor just here, add a paragraph that says "The [actions] must not be null."

Reply:
added